script.audio.profiles/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.audio.profiles" name="Audio Profiles" version="2.0.100" provider-name="pkscout, notoco, Regss">
+<addon id="script.audio.profiles" name="Audio Profiles" version="2.1.0~+matrix.1" provider-name="pkscout, notoco, Regss">
```

Comment:
Please use a valid version 2.1.0+matrix.1 otherwise the repository generator might break

Reply:
done.